### PR TITLE
Improved the documentation for torch.nn.functional.pad

### PR DESCRIPTION
**Pull request by bombs-kim**
- Fixed a few typos and grammar errors.
- Changed the sentences a bit.
- Changed the format of the tuples to be consistent with padding notations in the other places. For example, `ReflectionPad2d`'s dostring contains :math:`H_{out} = H_{in} + \text{padding\_top} + \text{padding\_bottom}`.

I also made sure that the generated html doesn't break.

### CODE REVIEW

**Review by ssnl — APPROVED** [review]
Thank you so much!

**Review by soumith — APPROVED** [review]
